src/lib/holocene/api-pagination.svelte

```diff
@@ -226,7 +226,7 @@
       <Input
         icon="search"
         id="api-pagination-search-input"
-        class="grow"
+        class="w-full grow sm:w-1/2"
```

Comment:
Looks like on big screens this goes back to columns instead of staying in a row ⬇️ 

<img width="1590" alt="Screenshot 2024-09-18 at 9 00 10 AM" src="https://github.com/user-attachments/assets/72c69cb5-c675-4741-aba8-454effd0faa4">

<img width="1327" alt="Screenshot 2024-09-18 at 9 01 08 AM" src="https://github.com/user-attachments/assets/1050bff1-e625-41b4-ac98-c0aa6d53a7ba">